crypto/chacha/asm/chachap10-ppc.pl

```diff
@@ -170,7 +170,9 @@ sub VSX_lane_ROUND_4x {
 .align	5
 .ChaCha20_ctr32_vsx_p10:
 	${UCMP}i $len,255
-	bgt 	ChaCha20_ctr32_vsx_8x
+	ble	.Not_greater_than_8x
```

Comment:
Thanks for addressing this..
As far as I can see this code was added 2 years old..
See https://github.com/openssl/openssl/commit/f596bbe4da779b56eea34d96168b557d78e1149a
Why is this only being flagged now? i.e. Why wouldn't this have been a problem for the original author?

How far away is the label? (it looks like conditional branches use 12 bits (4096)).

Reply:
This remote jump issue is only triggered when linking openssl to a larger application with a static library. 
For example [#21286](https://github.com/openssl/openssl/issues/21286)